<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showUI</class>
 <widget class="QMainWindow" name="showUI">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FourSearch</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border: 2px solid #dfdfdf;
background-color: rgba(0,0,0,0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="bird">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bird</string>
    </property>
   </widget>
   <widget class="QLabel" name="rightNUM">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>40</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="wrongNUM">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>40</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mAP1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mAP2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mAP3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mAP4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mAP2_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>mAP N=5</string>
    </property>
   </widget>
   <widget class="QLabel" name="mAP1_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>mAP N=1</string>
    </property>
   </widget>
   <widget class="QLabel" name="mAP3_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>mAP N=10</string>
    </property>
   </widget>
   <widget class="QLabel" name="mAP4_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>mAP N=50</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
